README.md

```diff
@@ -47,9 +47,16 @@ This extension contributes the following settings:
 * `ledgerDevTools.onboardingSeed`: Set the device quick onboarding 24-word Seed phrase.
 * `ledgerDevTools.dockerImage`: Set the Ledger developer tools Docker image.
 * `ledgerDevTools.additionalDepsPerApp`: Add dependencies for current app's functional tests (for instance 'apk add python3-protobuf').
+* `ledgerDevTools.keepContainerTerminal`: Indicates to keep the Terminal window opened after a successful Container Update.
+* `ledgerDevTools.ContainerUpdateTentatives`: Set the max number of Container Update tentatives.
```

Comment:
I would reword this setting to `ContainerUpdateRetries` (tentative does not have the proper meaning in english)

Reply:
Done